### DISCUSSION TITLE
Script removes and re-adds unchanged events

### DISCUSSION THREAD

**Discussion by bkgray** [derekantrican/GAS-ICS-Sync#353] [Q&A] [answered]
I installed version 5.7, and I'm doing testing with a calendar feed that has just two events.  The events have not been changed.

When the startSync() function runs from the trigger event, the emailed report says that the events were removed from the calendar and then they are re-added. 

When I run startSync() from the debugger, the events are reported as unchanged and are skipped (as expected).  

Any idea why events are seen as changed in one case but unchanged in the other?  

The feed is: webcal://sstx.myschoolapp.com/podium/feed/iCal.aspx?z=Hc8jGyzW%2fwwD4i6iY29vROTkDcvSmnT3kGiew8eAXb9Rb59vxrt6O4K%2fero8YKF%2b0mnX4AINSeH3p%2fY6YL6tcQ%3d%3d

**Comment by jonas0b1011001** [comment, marked as answer]
Hi, have a look at [this](https://github.com/derekantrican/GAS-ICS-Sync/issues/126#issuecomment-632060380), seems to still be the case.

**Reply by bkgray** [reply]
After reading your comment in the other thread, I looked at the execution logs for the script.  It does show that the deleted events have different IDs than the created ones.

THanks. for the quick response.
